Topics/Software_Tools/VSCode.md

```diff
@@ -167,6 +174,88 @@ Here are some quick hand-picked tips and tricks:
 
 For an extensive list of tips and tricks, visit the official [Visual Studio Code Tips and Tricks page](https://code.visualstudio.com/docs/getstarted/tips-and-tricks) to supercharge your VS Code experience.
 
+## Working with GitHub through VS Code
+
+Version control is a system that keeps track of changes to your files so that you can later recall a previous version (https://git-scm.com/book/en/v2/Getting-Started-About-Version-Control). GitHub is one of the most powerful such systems and is a go-to for many team projects like CSC301 because you can collaborate together, see the changes that have been made and you can rollback to a previous working version if something goes wrong.
+
+One of the most powerful use cases of VS Code is that it has integrated source control management and includes Git support. This means that whenever you are working on a remote repository on GitHub, you can clone, commit and push your code through VSCode conveniently and you can even track the changes you've made which can be very helpful. You can work with GitHub through the integrated terminal as well, and you can also directly work on it with the VS Code UI. This section will show how you can work on GitHub repositories with VS Code. This section is based on the VS Code Docs ["Using Git source control in VS Code"] (https://code.visualstudio.com/docs/sourcecontrol/overview) and ["Introduction to Git in VS Code"] (https://code.visualstudio.com/docs/sourcecontrol/intro-to-git)
+
+### Prerequisites
```

Comment:
Consider adding a link to the section on Git located somewhere in this repo

Reply:
Done!